controlpanel/frontend/jinja2/help.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+
+{% set page_title = "Help" %}
+
+{% block content %}
+  <div class="govuk-grid-row" id="reset-container">
+      <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Help</h1>
+
+    <p class="govuk-body">Common queries, support tasks and problems are
+    explained in the following videos.</p>
+
+    <p class="govuk-body">Should you still require help, please visit our
+    <a href="https://asdslack.slack.com/messages/C4PF7QAJZ#">support channel on Slack</a>, ask your question whilst remembering to tell us your
+    username and the name of any affected S3 buckets or applications on the
```

Comment:
I would probably be pedantic here and say "GitHub username" just to be sure users don't get confused with who knows what username 😬  (e.g. some users log into O365 using their email address as username IIRC so being more explicit may avoid confusion)